docs/src/content/en/docs/workflows/overview.mdx

```diff
@@ -164,22 +236,36 @@ Workflows can be run in two modes: start waits for all steps to complete before
 
 <Tabs items={["Start", "Stream"]}>
   <Tabs.Tab>
-Create a workflow run instance using `createRunAsync()`, then call `.start()` with `inputData` matching the workflow's `inputSchema`. The workflow executes all steps and returns the final result.
+Create a workflow run with `createRunAsync()`, then call `.start()` with `inputData` matching the workflow's `inputSchema`. To initialize workflow state, include `initialState` alongside `inputData`. The workflow runs all steps and returns the final result.
+
+To return state values in the result, set `includeState: true` in `outputOptions`.
```

Comment:
this is tricky

it does make sense to co-locate with start

but this doesn't apply unless you're using state, and most users aren't (it's new) and probably that won't change